news/2023/2023-12-12-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: How advances in technology affected the osu! community
+date: 2023-12-12 12:30:00 +0000
+---
+
+Let's walk back through the past 15 years of osu!, and how various new technologies affected the game and the community.
+
+![](/wiki/shared/news/2023-12-12-how-advances-in-technology-affected-the-osu-community/banner.jpg)
+
+Every now and then there's a new piece of tech that's slated to change the way games are played, whether it be a new input device, a new way to squeeze more FPS out of your PC, or a whole new way to experience games.
+
+While the osu! community has managed to stay unswayed as innovations such as virtual reality and augmented reality, cloud gaming & AI advancements cropped up due to them being inapplicable to osu!, there were still some that changed the way the game is played, or at least challenged the views on the most optimal way to play it.
+
+It's time to dive in and see how technology can impact the community, what the results of widespread adoption are, or could have been.
+
+## Graphics tablets (2007)
+
+osu! originally started out as an *[Osu! Tatakae! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan)* clone for PC, which was a game that was exclusive to the Nintendo DS, a console that came, and could be used with the included stylus pen. When [peppy](https://osu.ppy.sh/users/2) released osu!, it was only natural for players who already owned a graphics tablet to try it out in the so far unheard-of osu! context, and since the get-go it was considered a commodity input device that would more closely replicate the original Nintendo DS experience.
+
+Playing with a tablet is a completely different ball game compared to the mouse. Rather than being a direct upgrade by eliminating mouse drift with its absolute positioning, it brings up new challenges and questions regarding its optimal usage. A player figuring out their ideal pen grip, tablet area, and dragging or hovering play style have all been detrimental to increasing the learning curve, which is why some players choose to remain on the much more familiar and intuitive mouse.
+
+Over the years many top players and regular players started switching over to tablet in order to avoid issues associated with mice. Still, there has always been both mouse and tablet players that would compete at the very top of the leaderboards, as both are considered viable for competitive play, each with its own upsides and downsides.
```

Comment:
```suggestion
Over the years, many top players and regular players started switching over to tablet in order to avoid issues associated with mice. Still, there has always been both mouse and tablet players that would compete at the very top of the leaderboards, as both are considered viable for competitive play, each with its own upsides and downsides.
```